hazelcast/src/main/java/com/hazelcast/nio/serialization/PortableWriter.java

```diff
@@ -57,9 +57,20 @@
      * @param fieldName name of the field
      * @param value     utf string value to be written
      * @throws IOException in case of any exceptional case
+     * @deprecated for the sake of better naming. Use {@link #writeString(String, String)} instead.
```

Comment:
```suggestion
     * @Deprecated for the sake of better naming. Use {@link #writeString(String, String)} instead.
```

Reply:
The ones in the comments starts with lower letter. The annotations starts with capital letter